cpu-limits-low-perc-state-variables.tf

```diff
@@ -0,0 +1,62 @@
+variable "cpu_limits_low_perc_state_enabled" {
```

Comment:
If I understand correctly, you always either use the state version or the regular or the version that uses the state metrics API.

From a user perspective, perhaps I don't care that there are two different implementations of each of these metrics. How would you think about only using one set of monitor level variables for both the state version and the regular version?

The configuration would then look like this:

```hcl
module "my_cluster" {
  # ...

  use_state_metrics = true

  cpu_limits_low_state_warning = 95
  cpu_limits_low_state_critical = 100

  # ...
}
```

Switching to the regular version:


```hcl
module "my_cluster" {
  # ...

  use_state_metrics = false

  cpu_limits_low_state_warning = 95
  cpu_limits_low_state_critical = 100

  # ...
}
```